src/main/kotlin/org/koil/dashboard/org/OrgViews.kt

```diff
@@ -0,0 +1,26 @@
+package org.koil.dashboard.org
+
+import org.koil.auth.UserAuthority
+import org.koil.user.Account
+import org.koil.view.PaginatedViewModel
+import org.koil.view.ViewRenderer
+import org.springframework.data.domain.Page
+import java.util.*
+
+data class OrgIndexViewModel(val organizationName: String, val organizationSignupLink: UUID, val userName: String, val accounts: Page<Account>) :
+    PaginatedViewModel<Account>(accounts)
+
+data class OrgAccountDetailsViewModel(
+    val account: Account,
+    val updated: Boolean = false,
+    val emailAlreadyTaken: Boolean = false
+) {
+    val possibleAuthorities: List<Pair<UserAuthority, Boolean>> = UserAuthority.values().filter { it != UserAuthority.ADMIN }.map { authority ->
+        authority to account.authorities.map { it.authority }.contains(authority)
+    }
+}
+
+sealed class OrgViews<T>(override val template: String) : ViewRenderer<T> {
```

Comment:
I feel like it hasn't changed too much. The main thing seems to be we create them as an object rather than as a class that we create instances of. It seems very clean and to make sense- but maybe the old version was more different and I have just forgotten because I haven't worked in Koil for ages.